WeakAuras/Types.lua

```diff
@@ -1220,6 +1220,7 @@ Private.anchor_frame_types = {
   SCREEN = L["Screen/Parent Group"],
   UIPARENT = L["Screen"],
   PRD = L["Personal Resource Display"],
+  ACTION_BUTTON_SPELL = L["Action Button (Spell)"],
```

Comment:
Alternatively, this could be `ACTION_BUTTON` and the type (spell, item, equipment, 🤷🏻) could be an additional property selected via drop down. I didn't want to make that yet because I didn't actually add support for multiple types (just spell).